lib/mumuki/domain/factories/exercise_factory.rb

```diff
@@ -73,7 +81,7 @@
   factory :exercise, parent: :problem
 
   factory :indexed_exercise, parent: :exercise do
-    guide { create(:indexed_guide) }
+    indexed { true }
```

Comment:
I think it is a good idea to make all exercises factories _indexable_ on demand

Reply:
This is great!